src/nvim/ex_cmds_defs.h

```diff
@@ -176,6 +176,7 @@ typedef struct {
   int lockmarks;                        /* TRUE when ":lockmarks" was used */
   int keeppatterns;                     /* TRUE when ":keeppatterns" was used */
   bool noswapfile;                      /* true when ":noswapfile" was used */
+  bool browse;                          ///< TRUE to invoke file dialog
```

Comment:
It's Doxygen's format for documenting [members] of a struct, enum, etc.

[members]: http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#memberdoc